app/models/record.ts

```diff
@@ -89,4 +105,41 @@ export default abstract class Record {
         .update(fields);
     }
   }
-}
+
+  /**
+   * Validates and saves each record (using a single statement).  Throws an error if any
+   * record is not valid.  Records will be inserted and have their id, createdAt, and
+   * updatedAt fields set. If running SQLite, the id field will not be set as it only returns
```

Comment:
If it doesn't set the id field, that doesn't cause any problems for sqlite?

Reply:
It just doesn't set the id in memory, so it's only a problem if you need the id's. Thankfully in our case we're just inserting the objects and not using them in the code that follows.